sw/airborne/boards/bebop/board.c

```diff
@@ -33,6 +33,7 @@
 #include "mt9v117.h"
 #include "mt9f002.h"
 #include "mcu.h"
+#include BOARD_CONFIG
```

Comment:
why include BOARD_CONFIG here? wouldn't it be sufficient to include boards/bebop.h?